### ISSUE TITLE
Need help betting BBim running with 4.1 and IfcOpenShell 0.8

### ISSUE THREAD

**Issue by RickBrice** [IfcOpenShell/IfcOpenShell#4639]
I'm not really understanding the instructions in https://github.com/IfcOpenShell/IfcOpenShell/issues/4565.

I installed Blender 4.1, then installed the blenderbim-240402-py311-win.zip per the standard instructions https://docs.blenderbim.org/users/installation.html. 

Then I used the script for [Live development environment](https://docs.blenderbim.org/devs/installation.html). It runs without errors.

I fire up Blender, go to `Edit > Preferences` select `Add-ons` and scroll down the list to `System: BlenderBIM` and check the box.

Here is the error I get.

> Traceback (most recent call last):
>   File "C:\Program Files\Blender Foundation\Blender 4.1\4.1\scripts\modules\addon_utils.py", line 444, in enable
>     mod.register()
>   File "C:\Users\bricer\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\blenderbim\__init__.py", line 44, in register
>     blenderbim.bim.register()
>     ^^^^^^^^^^^^^^
> AttributeError: module 'blenderbim' has no attribute 'bim'

Other information that might be important:
- I have built IfcOpenShell, v0.8.0, from source
- I have Python v 3.12 installed.

Any ideas?
Thanks

**Comment by theoryshaw** [comment]
Could it be related to this?
https://github.com/IfcOpenShell/IfcOpenShell/issues/4635

The bat has recently changed.

**Comment by RickBrice** [comment]
I think the root cause problem was I had IfcOpenShell compiled for Python 3.12. I re-build it for Python 3.11 and it seems to work.

I tried opening the model from https://github.com/IfcOpenShell/IfcOpenShell/issues/4565#issuecomment-2068065492 and it crashed Blender. I used a very simplistic model I created and... wow... like magic, it worked

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/37087370/29b50b32-e5a9-4bef-8d59-db79d0f5d861)
